js/apps/system/_admin/aardvark/APP/frontend/js/views/documentView.js

```diff
@@ -339,8 +339,12 @@
         this.collection.saveEdge(this.colid, this.docid, $('#document-from').html(), $('#document-to').html(), model, callbackE);
       } else {
         var callback = function (error, data) {
-          if (error) {
-            arangoHelper.arangoError('Error', data.responseJSON.errorMessage);
+          if (error || data[0] && data[0].error) {
```

Comment:
next time please use parentheses here, to make clear the logical expression is intended to work like this.